vendor/engines/secure_rooms/app/controllers/secure_rooms_api/events_controller.rb

```diff
@@ -3,10 +3,28 @@ module SecureRoomsApi
   class EventsController < SecureRoomsApi::ApiController
 
     def create
-      Rails.logger.info "SecureRoomsApi Event: #{params}"
+      SecureRooms::Alert.create(alert_params)
       render nothing: true
     end
 
+    private
+
+    def alert_params
+      params.permit(
+        :additional_data,
+        :class_code,
+        :event_code,
+        :event_description,
+        :mac_address,
+        :message_id,
+        :message_time,
+        :message_type,
```

Comment:
Can we add a test for datetime parsing? Particularly that month/day are parsed USA-style rather than rest-of-the-world style?